app/views/ping_things/index.html.erb

```diff
@@ -61,10 +65,12 @@
             <div class="collapse word-break" id="collapseExample<%=index%>">
               <%= pt.signature %>
             </div>
-
           </td>
+          <td class="align-middle">&nbsp;<%= pt.application %></td>
```

Comment:
I don't think you need space (`&nbsp;`) in this column.
There's nothing before pt.application value in this column that we would want to separate by space.